util/wrap.pl

```diff
@@ -9,12 +9,15 @@
 my $there = canonpath(catdir(dirname($0), updir()));
 my $std_engines = catdir($there, 'engines');
 my $std_providers = catdir($there, 'providers');
+my $openssl_conf = catdir($there, 'apps/openssl.cnf');
```

Comment:
May I suggest `$std_openssl_conf`?

Reply:
Done in 32da9c15ed660e518a172d45c42c284f3627b470.